src/components/20-molecules/file-upload/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.9
+
+- Apply design-change suggestions from designers to allow coexistence of filename and per-file error message. (#1715)
```

Comment:
This is the only message needed, nobody cares about the rest. This was a "patch" update, which means, some bugs were fixed and so on. I would leave out the other two points. Patches are for bugfixes and refactorings, there is no need to repeat that.

Reply:
But the user could be interested which bugs exactly are fixed with the version. Especially if he reported the bug.